Base/QTGUI/qSlicerWebDownloadWidget.cxx

```diff
@@ -0,0 +1,148 @@
+/*==============================================================================
+
+  Program: 3D Slicer
+
+  Copyright (c) Kitware Inc.
+
+  See COPYRIGHT.txt
+  or http://www.slicer.org/copyright/copyright.txt for details.
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  This file was originally developed by Steve Pieper, Isomics Inc.
+  and was partially funded by NSF grant DBI 1759883
+
+==============================================================================*/
+
+// Qt includes
+#include <QDebug>
+#include <QDir>
+#include <QFileInfo>
+#include <QFileDialog>
+#include <QHBoxLayout>
+#include <QLabel>
+#include <QMessageBox>
+#include <QProgressBar>
+#include <QPushButton>
+#include <QVBoxLayout>
+#include <QWebEngineDownloadItem>
+
+// Slicer includes
+#include "qSlicerCoreApplication.h"
+#include "qSlicerDataDialog.h"
+#include "qSlicerWebDownloadWidget.h"
+
+// --------------------------------------------------------------------------
+qSlicerWebDownloadWidget::qSlicerWebDownloadWidget(QWidget *parent)
+  : QDialog(parent)
+{
+}
+
+// --------------------------------------------------------------------------
+void qSlicerWebDownloadWidget::handleDownload(QWebEngineDownloadItem *download)
+{
+
+// This code uses C++11 lambdas, which are only available when
+// Qt5 is used.  This code will never actually be called unless
+// QWebEngine is used, to it doesn't need to do anything in the
```

Comment:
SUPER minor: Only pointing this out because I happened to read this and saw this typo  (`to` --> `so`):
```suggestion
// QWebEngine is used, so it doesn't need to do anything in the
```